src/mds/MDCache.cc

```diff
@@ -12038,6 +12038,13 @@ void MDCache::show_subtrees(int dbl)
     return;
   }
 
+  if (!force_print && subtrees.size() > SUBTREES_COUNT_THRESHOLD &&
+      !g_conf()->subsys.should_gather<ceph_subsys_mds, 25>()) {
+    dout(dbl) << "number of subtrees = " << subtrees.size() << "; not "
```

Comment:
oops, `dout(dout(ceph::dout::need_dynamic(dbl)))`

Reply:
Made the change at [other instance](https://github.com/ceph/ceph/pull/26056/files#diff-2259d0a61fc0c9b2d2be3e40ade57303R12044) too.